### PR TITLE
[TASK] Allow multiple CC recipients

### PR DESCRIPTION
**Pull request by mhuber84**
With this change more than one CC recipient for emails can be configured as a comma separated list in the typoscript setup.

Closes: #457

### CODE REVIEW

File: Configuration/TypoScript/Main/setup.typoscript [hunk 1]
```diff
@@ -206,7 +206,7 @@ plugin.tx_femanager {
           subject = TEXT
           subject.data = LLL:EXT:femanager/Resources/Private/Language/locallang.xlf:emailCreateUserConfirmationSubject
 
-          # Set CC receivers
+          # Set CC receivers (comma separatet list for multiple CC receivers)
```
Comment (spoonerWeb) [inline Configuration/TypoScript/Main/setup.typoscript:209]: separatet => separated

For all changes below as well.

**Review by spoonerWeb — CHANGES_REQUESTED** [review]
Just a small typo but in many changes.

File: Configuration/TypoScript/Main/setup.typoscript (hunk 1 above)
Comment (mhuber84) [reply Configuration/TypoScript/Main/setup.typoscript]: Fixed

**Comment by mhuber84** [comment]
Can someone merge this one, please?

**Review by pixeldesu — APPROVED** [review]

**Comment by pixeldesu** [comment]
Since `develop` is outdated and changes should be made against `main` nowadays, I'll adjust the base branch of this PR.

Contributors can work in the branch, so I take care of the required adjustments and afterwards I'll merge it.

Thank you for your contribution!

**Comment by mhuber84** [comment]
Thank you @pixeldesu !

> Since develop is outdated and changes should be made against main
Can you switch the default branch in this repository to `main`? So new forks get the right base branch from the beginning.

**Pull request closed without merging**